### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by darxriggs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,84 +1,87 @@
...
+    </pluginRepositories>
 
-	<!-- get every artifact through maven.glassfish.org, which proxies all the artifacts that we need -->
-	<repositories>
-		<repository>
-			<id>repo.jenkins-ci.org</id>
-			<url>http://repo.jenkins-ci.org/public/</url>
-		</repository>
-	</repositories>
+    <distributionManagement>
```
Comment (oleg-nenashev) [inline pom.xml]: should be removed, not needed for recent Plugin POMs

**Comment by darxriggs** [comment]
Hi @vimil, @olivergondza and @oleg-nenashev, as some time already passed since this pull-request was created, I just wanted to check if it can be merged?

**Comment by darxriggs** [comment]
Let's try another ping.

**Comment by olivergondza** [comment]
The plugin appears unmaintained. Some of the contributors might consider [adopting it](https://wiki.jenkins.io/display/JENKINS/Adopt+a+Plugin).

**Comment by darxriggs** [comment]
I rebased on master to resolve the merge conflict in pom.xml.

**Comment by darxriggs** [comment]
@vimil could you have a look at this?

**Pull request merged**